### PR TITLE
Update the readme with changes on the issues policy

### PR DESCRIPTION
**Pull request by bidoubiwa**
We want to let users suggest features and let contributors implement them.

Additionally, I added a better explanation on what the mini dashboard is.

### CODE REVIEW

**Review by gmourier — COMMENTED** [review]
Seen it with @davelarkan, mentioning you so you can also review it 🙏

We don't necessarily want to block ideas and user feedback; it could benefit the cloud team to know what might be interesting regarding dashboard needs. However, the proposed features will not necessarily merge, and it will be a case-by-case study.

File: README.md [hunk 1]
```diff
@@ -19,8 +19,7 @@
 </p>
 <br/>
 
-<p align="center" style="font-weight:bold;" >Meilisearch's mini-dashboard</p>
-<p align="center">PRs and bug issues are welcome, but please do not submit any new feature requests 🙏</p>
+<p align="center" style="font-weight:bold;" >Meilisearch's mini-dashboard. A web-app served by the engine with a minimal search experience on your data.</p>
```
Comment (brunoocasali) [inline README.md:22]:
```suggestion
<p align="center" style="font-weight:bold;">Meilisearch's mini-dashboard. A web-app served by the engine with a minimal search experience on your data.</p>
```

**Review by brunoocasali — APPROVED** [review]
I agree with what @gmourier said. 

The more significant point is removing the section where we explicitly say that we don't accept feature requests, which may block a good communication flow with the users.
And, of course, those requests can be studied one by one (they are always like that) :)

**Comment by bidoubiwa** [comment]
bors merge

**Pull request merged**